app/models/telephone.rb

```diff
@@ -0,0 +1,7 @@
+class Telephone < ApplicationRecord
+  validates :telephone_type, presence: true, length: { maximum: 255 }
+  validates :ddi_number, presence: true, length: { maximum: 3 }
```

Comment:
There are some countries which ddi has four digits.
e.g.  República Dominicana and  Saint Martin